src/insexpand.c

```diff
@@ -1236,9 +1255,17 @@ ins_compl_build_pum(void)
 
     do
     {
+	// when completeopt include fuzzy option and leader is not null or empty
+	// set the cp_score for after compare.
+	if (compl_fuzzy_match == TRUE && compl_leader != NULL && lead_len > 0)
+	{
```

Comment:
The coding convention is to not use curly braces if there is a single statement in the body of a if statement.